src/services/FulfillmentService.js

```diff
@@ -31,10 +34,14 @@ class FulfillmentService {
         params: { slug },
         body: { type, value },
       } = req;
-
-      // TODO: get items by slug,type and value
-
-      return organizations;
+      const url = `${DOMAINS[node.env.NODE_ENV]}/rest/v1/fulfillments/services/${slug.zipCode}`;
```

Comment:
See my comment for the referrals pull request. The same applies here. We should discuss.